### PR TITLE
User Manual Updates for 0.6

### PR DESCRIPTION
**Pull request by vansia43**
Addresses all items listed in #740. 

Per #747, it sounds like CalDav docs will go in the next release or not right now.

### CODE REVIEW

**Review by diegogangl — APPROVED** [review]

**Comment by diegogangl** [comment]
Thanks for the PR! We should have some docs for caldav at release though

File: docs/user_manual/C/gtg-plugins.page [hunk 1]
```diff
@@ -39,6 +40,9 @@
 <p><em>Export and Print</em></p>
 <p>Exports the tasks in the current view into a variety of formats.</p>
 
+<p><em>Gamify</em></p>
+<p>Gamify adds a game aspect to your GTG workflow.</p>
```
Comment (Neui) [inline docs/user_manual/C/gtg-plugins.page:44]: Maybe a short mention what "game aspect" means here? Like "(like task completion streaks)".

**Review by Neui — APPROVED** [review]
Changes are OK.

I am wondering whenever we should be more "distro-neutral" in the `set_date_task_browser.png` image (where you can see the background image). Additionally, if replaced with solid colour (or even transparency), then we might save some kilobytes because of better image compression (maybe run something like `optipng` on those 2 changed images, to get the most out of it?)

File: docs/user_manual/C/gtg-plugins.page (hunk 1 above)
Comment (vansia43) [reply docs/user_manual/C/gtg-plugins.page]: Good point! Added a bit more of a description for this.

**Comment by vansia43** [comment]
> Thanks for the PR! We should have some docs for caldav at release though

I made the 2 requested changes...

As for CALDAV. I hear you. I tried to grab the latest and greatest code to test it out, but I am having trouble getting it to work on my end though, so I can't really come up with any docs for it. >.<

**Comment by diegogangl** [comment]
Maybe it'd be easier to have @jaesivsm write some notes, and then we can put the docs together from that

**Review by nekohayo — APPROVED** [review]
It looks good to me too, thank you for paying attention to these details!

**Comment by diegogangl** [comment]
Merged, thanks! We can work on the caldav docs on another branch

**Comment by Shahin-rmz** [comment]
Hello, I have followed #407 issue and end up being here, where I can see, the caldav sync feature is already done, and is waiting for documentation, until in next release it implemented.
There is a question though, how can we update and/or install the plugin in order to be able to use it?
Thanks

**Comment by dmp1ce** [comment]
With the latest code, are you getting any errors? I was missing the caldav python module which I only noticed because of a warning on the CLI.

**Comment by diegogangl** [comment]
Good point, we also need to include that in the docs (at least for now).

**Comment by Shahin-rmz** [comment]
> With the latest code, are you getting any errors?
I haven't got any error, actually I don't know how to use CalDave. GTG is updated but don't know how to connect to caldav.

**Comment by Neui** [comment]
> I haven't got any error, actually I don't know how to use CalDave. GTG is updated but don't know how to connect to caldav.

Also took me a while, you need to have `caldav` python packgae installed (I did it via `pip3 install caldav`), then go to the menu top left → Synchronization, then on the plus bottom left. The right side should show a "Add backend" thing with caldav automatically being selected with some options and an OK bottom right.

**Comment by Shahin-rmz** [comment]
> Also took me a while...
Thanks so much for the fast answer.
I have actually installed caldav now.
image
But after that, the sync button is still deactivated.
image

**Comment by Shahin-rmz** [comment]
not sure if here is the right place to talk about installation AND if I missed any dependencies.
App works fine but with sync button deactivated.

**Comment by Neui** [comment]
> App works fine but with sync button deactivated.

What version are you on? 5.0 or the git version? You need the current git version to make it selectable.

**Comment by Shahin-rmz** [comment]
image
I have version 0.5, not 5!
I'll check for git version.
let's hope I can find it within documentation. :)

**Comment by Shahin-rmz** [comment]
image
I have installed flatpak version, pulled git version and tried to update it and that was the result.

**Comment by diegogangl** [comment]
Hi @Shahin-rmz , did you run GTG with the `./launch` command? Was the flatpak version from flathub (that one is 0.5)?

**Comment by Shahin-rmz** [comment]
Hi @diegogangl , you are right.
flatpack was from flathub. and I didn't run it via ./launch command.
just double click.

**Pull request merged**